### PR TITLE
Fixes #73 Add documentation to class LabelEncoder and to method infer…

### LINKED ISSUE #73: Add Documentation for LabelEncoder
It's difficult to understand what `LabelEncoder` does. I have tried  ooking at the method `inferLabelEncoder` in the `CsvLoader.scala`, to be specific the line 
```
encoder(name)(featureValue) = encoder(name).size
```
But it's not yet clear what `LabelEncoder` is being used for. 

Having  a comment documentation would really help here.

### PR DESCRIPTION
**Pull request by ashwinbhaskar**
…LabelEncoding

##### Includes

- [x] Documentation

### CODE REVIEW

File: src/main/scala/io/picnicml/doddlemodel/data/CsvLoader.scala [hunk 1]
```diff
@@ -51,6 +51,13 @@ object CsvLoader {
     FeatureIndex(featureNames, featureTypes, featureNames.indices.toList)
   }
 
+  /** Transforms Feature Values with non-numerical type (Eg Categorical types) to a numerical value
```
Comment (inejc) [inline src/main/scala/io/picnicml/doddlemodel/data/CsvLoader.scala]: Maybe we should avoid using the word `transforms` here as this function actually creates the `LabelEncoder` which is then used to do the transformation that you described? I believe something like `creates a label encoder for a given dataset...` would better describe the functionality.

**Comment by ashwinbhaskar** [comment]
@ashwinbhaskar no worries:) I have made the changes that you pointed out and pushed

**Comment by ashwinbhaskar** [comment]
@inejc Also the build seems to have failed. I do not have permission to trigger it again. Can you trigger it again, please?

**Comment by inejc** [comment]
All builds from forked repos are failing because circleci secrets are not copied there (i.e. the `Codacy` token is missing). I need to find a workaround for that.

**Pull request merged**